### PR TITLE
Drop inf & -inf from TrialValueNumber

### PR DESCRIPTION
**Pull request by porink0424**
## Contributor License Agreement

This repository (``optuna-dashboard``) and [Goptuna](https://github.com/c-bata/goptuna) share common code.
This pull request may therefore be ported to Goptuna.
Make sure that you understand the consequences concerning licenses and check the box below if you accept the term before creating this pull request.

* [x] I agree this patch may be ported to [Goptuna](https://github.com/c-bata/goptuna) by other Goptuna contributors.

## What does this implement/fix? Explain your changes.

Drop inf & -inf from TrialValueNumber

### CODE REVIEW

**Comment by nabenabe0928** [comment]
It seems the following files need the changes:

- [ ](optuna_dashboard/_serializer.py)
- [ ](optuna_dashboard/ts/components/GraphContour.tsx)
- [ ](optuna_dashboard/ts/components/GraphIntermediateValues.tsx)
- [ ](optuna_dashboard/ts/components/GraphParetoFront.tsx)
- [ ](optuna_dashboard/ts/components/GraphRank.tsx)
- [ ](optuna_dashboard/ts/components/TrialFormWidgets.tsx)
- [ ](optuna_dashboard/ts/components/TrialTable.tsx)
- [ ](optuna_dashboard/ts/dominatedTrials.ts)
- [ ](optuna_dashboard/ts/trialFilter.ts)
- [ ](optuna_dashboard/ts/types/index.d.ts)
- [ ](standalone_app/src/components/PlotHistory.tsx)
- [ ](standalone_app/src/components/PlotImportance.tsx)
- [ ](standalone_app/src/components/PlotIntermediateValues.tsx)
- [ ](standalone_app/src/components/TrialTable.tsx)
- [ ](standalone_app/src/sqlite3.ts)
- [ ](standalone_app/src/types/index.d.ts)

**Comment by c-bata** [comment]
@nabenabe0928 Thank you for your review! The codebases for the standalone app (which is internally used from the VS Code extension) are separated with the codebases for the Python package. So could you only check the files under the `standalone_app` directory?

File: standalone_app/src/types/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 declare const IS_VSCODE: boolean
 
-type TrialValueNumber = number | "inf" | "-inf"
+type TrialValueNumber = number
 type TrialIntermediateValueNumber = number | "inf" | "-inf" | "nan"
```
Comment (nabenabe0928) [inline standalone_app/src/types/index.d.ts:4]: Do we also need to change here as well?
And `standalone_app/src/components/PlotIntermediateValues.tsx`?

File: standalone_app/src/sqlite3.ts [hunk 2]
```diff
@@ -389,11 +389,11 @@ const getTrialIntermediateValues = (
           step: vals[0],
           value:
             vals[2] === "INF_NEG"
-              ? "-inf"
```
Comment (nabenabe0928) [inline standalone_app/src/sqlite3.ts:392]: Do we also need to change Line 221--Lline223, which is for multi-objective values?

**Review by nabenabe0928 — APPROVED** [review]
Thank you for the change, LGTM"(

**Pull request merged**